runtime/ftplugin.vim

```diff
@@ -30,9 +30,10 @@ augroup filetypeplugin
       for name in split(s, '\.')
         " Load Lua ftplugins after Vim ftplugins _per directory_
         " TODO(clason): use nvim__get_runtime when supports globs and modeline
-        exe printf('runtime! ftplugin/%s.vim ftplugin/%s.lua', name, name)
-        exe printf('runtime! ftplugin/%s_*.vim ftplugin/%s_*.lua', name, name)
-        exe printf('runtime! ftplugin/%s/*.vim ftplugin/%s/*.lua', name, name)
+        exe printf('runtime! ftplugin/%s.vim ftplugin/%s.lua
+                           \ ftplugin/%s_*.vim ftplugin/%s_*.lua
+                           \ ftplugin/%s/*.vim ftplugin/%s/*.lua',
+                           \ name, name, name, name, name, name)
```

Comment:
Shouldn't this be?

```suggestion
        exe printf('runtime! ftplugin/%s.vim ftplugin/%s_*.vim ftplugin/%s/*.vim
                           \ ftplugin/%s.lua ftplugin/%s_*.lua ftplugin/%s/*.lua
                           \ name, name, name, name, name, name)
```

Reply:
Debatable; would have to test.

Reply:
Otherwise I don't think this PR does anything.

Reply:
Well, it does _something_ (right), according to the person who tested it.

Reply:
Yeah, sorry, I realize now. What I said before wasn't correct.